contrib/opencensus-ext-azure/opencensus/ext/azure/log_exporter/__init__.py

```diff
@@ -30,6 +31,8 @@
 from opencensus.ext.azure.common.transport import TransportMixin
 
 logger = logging.getLogger(__name__)
+if 'is_exporter_thread' not in RuntimeContext.snapshot().keys():
```

Comment:
I guess there is a better place for this instead of the `log_exporter`.
Going forward in OpenTelemetry we're trying to remove the requirement of having to register slots.

Reply:
Right. Instead of checking and changing the RuntimeContext for every exporter, we use a slot in the execution context now instead and simply import the module in each exporter.